frontend/util/services-inputs.component.tsx

```diff
@@ -69,11 +67,31 @@ export default React.forwardRef<ServicesRef, IntakeServicesInputsProps>(
 );
 
 const css = `
-& h1 {
-  font-size: 1.8rem;
-}
+	& h3 {
```

Comment:
tabs -> spaces throughout this template string. Please configure your IDE to always use spaces. https://stackoverflow.com/questions/29972396/how-to-customize-the-tab-to-space-conversion-factor